src/components/MainAppHeader.js

```diff
@@ -61,24 +58,22 @@ class MainAppHeader extends Component {
     const { classes } = this.props;
 
     return (
-      <div className={ classes.grow }>
-        <AppBar className={ classes.root }>
-          <Toolbar className={ classes.toolBar }>
-            <MenuIcon
-              fallBackClasses="bm-burger-button"
-              onClick={ this.handleMenu() }
-              testID="open-menu-icon"
-            />
-            <img src={ headerLogoPng } srcSet={ headerLogoSvg } className={ classes.headerLogo }/>
-            <CartIcon
-              fallBackClasses="shopping_cart_link"
-              onClick={ this.openCart() }
-              testID="cart-icon"
-              cartContent={1}
-            />
-          </Toolbar>
-        </AppBar>
-      </div>
+      <AppBar className={ classes.root }>
+        <Toolbar className={ classes.toolBar }>
+          <MenuIcon
+            fallBackClasses="bm-burger-button"
+            onClick={ this.handleMenu() }
+            testID="open-menu-icon"
+          />
+          <img src={ headerLogoPng } srcSet={ headerLogoSvg } className={ classes.headerLogo }/>
+          <CartIcon
+            fallBackClasses="shopping_cart_link"
+            onClick={ this.openCart() }
```

Comment:
I realize this currently doesn't do anything, but should this be `onClick={ this.openCart }` are you planning on having that method return a function? Same with the `MenuIcon` onClick handler above.

Reply:
I haven't implemented the code behind it, but will do when all components are done and the pages are build.